great_expectations/core/util.py

```diff
@@ -69,6 +69,12 @@
 
 _SUFFIX_TO_PD_KWARG = {"gz": "gzip", "zip": "zip", "bz2": "bz2", "xz": "xz"}
 
+TEMPORARY_EXPECTATION_SUITE_NAME_PREFIX: str = "tmp"
+TEMPORARY_EXPECTATION_SUITE_NAME_STEM: str = "suite"
```

Comment:
Do we need these as constants or can we just directly use them in the pattern? I'd like to reduce the number of variables in the global namespace.

Reply:
@cdkini These constants are helpful in order to keep different utilities in sync.  I had the same back-and-forth and felt that having them was more beneficial than otherwise.  Thanks.